vars/CKAN-Stack.var.yml

```diff
@@ -6,7 +6,7 @@ ProductionAnalyticsId: "{{ lookup('aws_ssm', '/config/CKAN/GaIdProduction', regi
 
 solr7: "http://archive.apache.org/dist/lucene/solr/7.7.2/solr-7.7.2.zip"
 ckan_tag: "ckan-2.8.8-qgov.5"
-ckan_qgov_branch: "qgov-master-2.8.8"
+ckan_qgov_branch: "qgov-master-2.9.5"
```

Comment:
ckan-2.9.5-qgov.2 to qgov-master-2.9.5 ?

Reply:
qgov-master-2.9.5 is a branch rather than a tag. But currently they're the same commit.